samples/node-headless-ssr-proxy/README.md

```diff
@@ -56,3 +56,10 @@ More info on this setup can be found [here](https://jss.sitecore.com/docs/fundam
 - Ensure that `debug: false` in `config.js`. Console output will cause terrible scaling.
 - Customise `error.html` in case your app throws HTTP 500 errors.
 - Load test the proxy prior to launch to ensure proper performance levels for your needs.
+
+3. **Node-level output caching:**
+Node hedless ssr proxy uses Node-level output caching by default , implementation of the caching you can find in cacheMiddleware.js file , also there is `server.use(cacheMiddleware(..))` in index.js.If you don't need Node-level output caching , please delete this line.
+
+    ###Potential limitations of using node-level output caching:
+    - Security: It could be a case if users with different roles see the same output, in this case, you need to customize `cacheMiddleware` , f.e. add to cache key user identifier.
+    - Personalization : Potentially you have a limitation of using using output caching with personalization , need to customize `cacheMiddleware` if neeeded.
```

Comment:
@anton-ovch-sitecore Seems that it should not be here, as in `README.MD` usually described process of running and configuring the application. Probably can delete it